index.bs

```diff
@@ -2857,7 +2868,8 @@ Given an [=attribution trigger=] |trigger| and [=attribution source=] |sourceToA
 Given an [=attribution rate-limit record=] |newRecord|:
 
 1. Let |max| be [=max source reporting origins per rate-limit window=].
-1. If |newRecord|'s [=attribution rate-limit record/scope=] is "<code>[=rate-limit scope/attribution=]</code>", set |max| to
+1. If |newRecord|'s [=attribution rate-limit record/scope=] is "<code>[=rate-limit scope/event-attribution=]</code>" 
```

Comment:
This is a limit on distinct reporting origins. What is the reasoning for splitting this into event- and aggregate-levels?